cmd/create/config.go

```diff
@@ -90,6 +95,22 @@ func NewConfigCmd(tracker *analytics.Tracker) *cobra.Command {
 			// Init packages.
 			execx.Debug = debug
 
+			// Validate path.
+			if _, err := os.Stat(furyctlPath); err == nil {
+				absPath, err := filepath.Abs(furyctlPath)
+				if err != nil {
+					return fmt.Errorf("%w: error while getting absolute path %v", ErrConfigCreationFailed, err)
+				}
+
+				p := filepath.Dir(absPath)
+
+				return fmt.Errorf(
+					"%w: a furyctl.yaml configuration file already exists in %s, please remove it and try again",
```

Comment:
let's not hardcode `furyctl.yaml`, the name of the file could be anything the user wants

```suggestion
					"%w: a configuration file already exists in %s, please remove it and try again",
```